src/Workspaces/Core/Portable/Workspaces.csproj

```diff
@@ -268,6 +268,7 @@
     <InternalsVisibleTo Include="Microsoft.VisualStudio.VisualBasic.Repl" />
     <InternalsVisibleTo Include="Roslyn.VisualStudio.Setup" />
     <InternalsVisibleTo Include="Roslyn.VisualStudio.DiagnosticsWindow" />
+    <InternalsVisibleTo Include="Roslyn.VisualStudio.RemoteHostClientMock" />
```

Comment:
Is this 'Mock' purely for testing purposes?  It's not clear why it needs to be its own assembly.